### PR TITLE
dashboard seach fix

### LINKED ISSUE #5921: Using search bar on PL website leads to error page
Hey guys!

Multiple times today I've had issues reaching the search page. When I go into the search bar, type my query, and hit enter, I'm redirected here:

Screen Shot 2019-06-18 at 2 13 09 PM

Happens in Chrome & Safari before and after clearing cache. 

Same error page from publiclab.org/search and from any website page via the search option in the NavBar.

### PR DESCRIPTION
**Pull request by gauravano**
Fixes #5921 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by gauravano** [comment]
Hey @jywarren, one test is failing. I think, change is required in one more method. 

Screenshot_2019-06-19-18-48-41-51

**Comment by jywarren** [comment]
:-) this is like a very Jeff type of comment 😂 

OK I'll adjust that file!

**Comment by jywarren** [comment]
Done!

File: test/functional/search_controller_test.rb [hunk 1]
```diff
@@ -54,7 +54,7 @@ class SearchControllerTest < ActionController::TestCase
   test "search tags at /search/tags/awesome" do
     get :tags, params: { query: 'awesome' }
     assert_response :success
-    assert_equal node_tags(:awesome).name, assigns(:tags).first
+    assert_equal node_tags(:awesome).tag, assigns(:tags).first
```
Comment (jywarren) [inline test/functional/search_controller_test.rb]: Oops, we need tag, not node_tag... fixed

**Comment by gauravano** [comment]
Great!

**Pull request merged**